crates/api_server/Cargo.toml

```diff
@@ -19,8 +19,10 @@ anvil_zksync_common.workspace = true
 
 zksync_types.workspace = true
 zksync_web3_decl.workspace = true
+zksync-telemetry.workspace = true
```

Comment:
Nit: We use `_` rather than `-` in crate names, probably makes sense to rename it. Not urgent tho.

Reply:
I see, better to rename now, less to refactor later.

Reply:
Done